cabal-testsuite/PackageTests/TestSuiteTests/ExeV10/cabal-with-hpc.multitest.hs

```diff
@@ -9,8 +9,8 @@ import qualified Distribution.Verbosity as Verbosity
 import Test.Cabal.Prelude
 
 main = cabalTest $ do
-    skipIf "osx" =<< isOSX -- TODO: re-enable this once the macOS CI
-                           -- issues are resolved, see discussion in #4902.
+    skipIf "osx" isOSX -- TODO: re-enable this once the macOS CI
+                       -- issues are resolved, see discussion in #4902.
```

Comment:
This maybe can be re-enabled. I won't touch it just yet as I'm fixing Windows issues now.